bin/spark-connect-scala-client.sc

```diff
@@ -0,0 +1,15 @@
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.SparkSession
+
+val conStr = if (sys.env.contains("SPARK_REMOTE")) sys.env("SPARK_REMOTE") else ""
```

Comment:
Feature request, can we try to configure this using a `--connectionString` argument?

Reply:
This feature request is a bit hard :D I cannot figure out a way to pass the args to the program rather than scala repl. 

I think it relates '-i'. We need '-i' to keep the repl running, then the repl thinks all args are to the repl rather than the scala script.